age-calculator/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<div class="container">
+		<div class="input">
+			<input type="date" id="date">
+			<button onclick="ageCalc()">Calculate</button>
+		</div>
+		<div class="output">
+			<div>
+				<span id="years">
+					
+				</span>
+				<p>YEARS</p>
+			</div>
+			<div>
+				<span id="months">
+					
+				</span>
+				<p>MONTHS</p>
+			</div>
+			<div>
+				<span id="days">
+					
+				</span>
+				<p>DAYS</p>
+
+
+
+		</div>
+	</div>
+</div>
+	<script type="text/javascript" src="script.js"></script>
+</body>
+</html>
```

Comment:
Add new line at end of file

Reply:
Use a html beautifier, and increase readability. 
Here use this link - https://codebeautify.org/htmlviewer